src/Builders/ServersBuilder.php

```diff
@@ -13,7 +15,26 @@ class ServersBuilder
     public function build(array $config): array
     {
         return collect($config)
-            ->map(static fn(array $server) => Server::create()->url($server['url']))
+            ->map(static function (array $server) {
+                $variables = collect(Arr::get($server, 'variables'))
+                    ->map(function (array $variable, string $key) {
+                        $tmp = ServerVariable::create($key)
+                            ->default(Arr::get($variable, 'default'))
+                            ->description(Arr::get($variable, 'description'));
+                        if (is_array(Arr::get($variable, 'enum'))) {
+                            // exist
```

Comment:
This comment does not add anything, it should probably be removed.

Reply:
+ `if` can be refactored to `->when(is_array...`

Reply:
I don't know how to refactor using `->when(...` , if you don't mind.